mm/page_alloc.c

```diff
@@ -111,7 +113,42 @@ EXPORT_SYMBOL(node_states);
 /* Protect totalram_pages and zone->managed_pages */
 static DEFINE_SPINLOCK(managed_page_count_lock);
 
+#ifndef CONFIG_LIB
 unsigned long totalram_pages __read_mostly;
+#endif
+
+
+static void print_buddy_freelist(void)
```

Comment:
if this isn't used and not needed, remove it.
or is it possible to move to other code in arch/lib as well ?